templates/interactive/analysis_request_detail.html

```diff
@@ -46,7 +51,7 @@ <h1 class="text-3xl tracking-tight break-words sm:text-4xl font-bold text-slate-
     <dl class="border-t border-slate-200 sm:divide-y sm:divide-slate-200">
       {% #description_item title="Created at" %}
         {{ analysis_request.created_at }}
-        <time datetime="{{ analysis_request.created_at|date:"Y-m-d H:i:sO" }}"></time>
+        <time datetime='{{ analysis_request.created_at|date:"Y-m-d H:i:sO" }}'></time>
```

Comment:
Should we be using single quotes here normally?

Reply:
Oh I think was just my editor not formatting the file correctly (thank you incorrect VSCode plugin). I will switch this back.